exporter/signalfxexporter/internal/correlation/correlation_test.go

```diff
@@ -86,6 +86,10 @@ func TestTrackerStart(t *testing.T) {
 			} else {
 				require.NoError(t, err)
 			}
+
+			if tracker != nil {
+				assert.NoError(t, tracker.Shutdown(context.Background()))
+			}
```

Comment:
How tracker can be nil at this point?

Reply:
Good catch, I don't think it can. I don't remember my thought process here, but it may have been concern about an error on start. If `tracker.Start` is called and `tracker` is nil, that would panic anyways, so there's no need to check below it in the test.

The test is passing without this check, so I'll remove it.